serving/src/test/java/ai/djl/serving/ModelServerTest.java

```diff
@@ -160,6 +160,9 @@ public void beforeSuite() throws IOException {
         String engineCacheDir = Utils.getEngineCacheDir().toString();
         System.setProperty("DJL_CACHE_DIR", "build/cache");
         System.setProperty("ENGINE_CACHE_DIR", engineCacheDir);
+
+        // TODO Remove when removing the temporary feature flag
+        System.setProperty("ENABLE_ADAPTER_SUPPORT", "true");
```

Comment:
How about using `ENABLE_ADAPTER_PREVIEW_SUPPORT` to explicitly state preview?

Reply:
Renamed to `ENABLE_ADAPTERS_PREVIEW`